### PR TITLE
Updated documantation and example

### PR DESCRIPTION
**Pull request by marcin-krolik**
Updates documentation and examples

Summary of changes:
- removed SNAP_PATH
- added example task
- capitalized Snap

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,27 +22,34 @@ It's used in the [snap framework](http://github.com/intelsdi-x/snap).
...
+Fork https://github.com/intelsdi-x/snap-plugin-collector-etcd
 Clone repo into `$GOPATH/src/github.com/intelsdi-x/`:
 
 ```
-$ git clone https://github.com/intelsdi-x/snap-plugin-collector-etcd.git
+$ git clone https://github.com/<yourGithubID>/snap-plugin-collector-etcd.git
 ```
 
-Build the plugin by running make within the cloned repo:
+Build the snap etcd plugin by running make within the cloned repo:
```
Comment (kindermoumoute) [inline README.md]: *Snap

File: README.md [hunk 2]
```diff
@@ -130,56 +137,40 @@ And the raw metrics:
 ```
 
 ### Examples
-This is done from the snap directory.
+Example of running snap etcd collector and writing data to file.
```
Comment (kindermoumoute) [inline README.md]: *Snap

File: README.md [hunk 3]
```diff
@@ -130,56 +137,40 @@ And the raw metrics:
...
 
-In one terminal window, start the snap daemon (in this case with logging set to 1 and trust disabled):
-```
-$ $SNAP_PATH/bin/snapd -l 1 -t 0
-```
+Ensure [snap daemon is running](https://github.com/intelsdi-x/snap#running-snap):
```
Comment (kindermoumoute) [inline README.md]: *Snap

File: README.md [hunk 4]
```diff
@@ -130,56 +137,40 @@ And the raw metrics:
...
+* initd: `service snap-telemetry start`
+* systemd: `systemctl start snap-telemetry`
+* command line: `snapd -l 1 -t 0 &`
 
-In another terminal window:
-Load etcd plugin
+Download and load snap plugins:
```
Comment (kindermoumoute) [inline README.md]: *Snap

**Comment by marcin-krolik** [comment]
@kindermoumoute I think the reason for trigger was I pushed `update/docs` directly to this repo. I noticed this mistake and removed that branch.

**Comment by kindermoumoute** [comment]
LGTM

**Pull request merged**